Update look of boxes for consistency (ErrorHandler, Updater, ...)

tsteur opened the pull request:
We removed the border-radius and another border from the widgets in dashboard and also started using the card design in some places so thought might be consistent to use no border there as well

UI tests: http://builds-artifacts.piwik.org/ui-tests.no_border_radius_in_boxes/13715.7/

eg

http://builds-artifacts.piwik.org/ui-tests.no_border_radius_in_boxes/13715.7/CoreUpdaterDb_updated
http://builds-artifacts.piwik.org/ui-tests.no_border_radius_in_boxes/13715.7/UIIntegrationTest_db_connect_error
http://builds-artifacts.piwik.org/ui-tests.no_border_radius_in_boxes/13715.7/UIIntegrationTest_widgetize_apidisallowed

FYI: I noticed there is the background-color #f0f0f0 used but in Piwik itself we often use #f2f2f2. I didn't change it as I thought this color might be used in the future instead

tsteur committed:
Remove border radius in boxes for consistency

Want to see the screenshot tests before making any decision.

tsteur committed:
also adjust the shadow and border of content for consistency

mnapoli commented:
Why did you remove the radius? It doesn't look really good IMO.

mnapoli commented on plugins/Morpheus/stylesheets/simple_structure.css:
```diff
@@ -16,17 +16,18 @@ body#simple  {
-    box-shadow: 0 1px 2px 0 #ccc;
+    -webkit-box-shadow: 0 1px 6px 0 #ccc;
+    -moz-box-shadow: 0 1px 6px 0 #ccc;
+    box-shadow: 0 1px 6px 0 #ccc;
```
No need for the aliases.

tsteur commented:
as described I changed it for consistency to reuse the card design there as well. I kinda like it better since it looks like this as well in Dashboard, Visitor Log, Sites Manager, Marketplace (kinda), ... and we use border-radius less and less often. I updated the links to the diffs

mnapoli commented:
yes, as I said one needs to be aware of the pr created and merged during the night to understand where this comes from.

tsteur committed:
remove the border radius aliases

tsteur commented:
> yes, as I said one needs to be aware of the pr created and merged during the night to understand where this comes from.

We're making improvements on merging PRs faster :)

mattab commented:
@mnapoli removing borders is a bit of an experiment in simplifying further and making style consistent... it was not so clear whether to do it or not (or with a 1 or 2px border radius), but I'm keen to try without and see where it leads 

I think this PR is the last Design change for Piwik 2.14.0 - we did really well at refreshing the design in this short time :+1: 

our next big step in terms of improving the design will be the Left Menu #7589 (and maybe review the top menu too).
